### PR TITLE
Updated setup instructions

### PR DESCRIPTION
**Pull request by jcohen02**
This update revises the setup instructions for Windows 10, macOS and Linux. It also moves the `docker-intro.zip` file into the `files` directory from where it can be accessed through a link in the rendered version of the setup information.

The main changes have been in relation to Windows 10 where Docker Toolbox is now deprecated and Windows 10 Home now supports docker via the WSL2 backend.

While Docker Toolbox is now deprecated and doesn't seem to be downloadable from the Docker install instructions, the final release can still be downloaded from the [its GitHub repository](https://github.com/docker/toolbox/releases).

If people are using an old system and can't install the latest Docker Desktop, using Toolbox might provide a way for them to be able to work through this material when they otherwise couldn't, so for training purposes, this might be OK. At the same time, I wonder if the fact that Docker has moved on significantly from the version used in the last release of Toolbox means that there are potential security issues with installing this old version? As such, I'm wondering if the short paragraphs highlighting Docker Toolbox in these additions should actually be removed?

The updates provided here are ported from modifications made for the running of a [recent course](https://github.com/EPCCed/2020-12-08-Containers-Online) covering this material.

### CODE REVIEW

**Review by ChristinaLK — APPROVED** [review]
this looks great! I like the expansion of these instructions and moving the zip file.

File: setup.md [hunk 1]
```diff
@@ -7,33 +7,51 @@ Please seek help at the start of the lesson if you have not been able to establi
...
 
-If you have Windows 10 Pro Edition:
- - First try to install the [Docker Desktop (Windows)](https://hub.docker.com/editions/community/docker-ce-desktop-windows), or **failing that**;
- - Install the [Docker Toolbox (Windows)](https://docs.docker.com/toolbox/toolbox_install_windows/).
+#### **Microsoft Windows**
+**You must have admin rights to run docker!** Some parts of the lesson will work without running as admin but if you are unable to `Run as administrator` on your machine some elements of this workshop might not work as described.
+
+On Windows 10, you will need to install [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). 
+
+_Previously Windows 10 Home users needed to install an alternative version of Docker - [Docker Toolbox](https://docs.docker.com/docker-for-windows/docker-toolbox/) - however, this is now deprecated. Docker Desktop for Windows can now also be installed on Windows 10 Home Edition_
```
Comment (sstevens2) [inline setup.md:25]: I suggest we remove this line.  I don't think people installing for the first time will need to know this.

File: setup.md [hunk 2]
```diff
@@ -7,33 +7,51 @@ Please seek help at the start of the lesson if you have not been able to establi
...
-[Docker Introduction Github](https://github.com/carpentries-incubator/docker-introduction/). 
+Download the [`docker-intro.zip`]({{ page.root }}/files/docker-intro.zip) file. _This file can alternatively be downloaded from the `files` directory in the [docker-introduction GitHub repository](https://github.com/carpentries-incubator/docker-introduction/)_.
 
 Move the downloaded file to your Desktop and unzip it. It should unzip to a folder called `docker-intro`. 
 
-### Software to install
-Unfortunately, in many common situations installing Docker on your laptop will not straightforward if you do not have a large amount of technical experience. We have helpers on hand that have worked their way through the install process but be prepared for some troubleshooting.
+### Software to install: Docker
+
+Docker can be installed on Windows 10, macOS and Linux. However, in some cases, installing Docker on your laptop may, unfortunately, not be completely straightforward if you do not have significant technical experience. We have helpers on hand that have worked their way through the install process but be prepared for some troubleshooting.
```
Comment (sstevens2) [inline setup.md:16]:
```suggestion
Docker can be installed on Windows 10, macOS and Linux. However, in some cases, installing Docker on your laptop may, unfortunately, not be completely straightforward. We have helpers on hand during the workshop that have worked their way through the install process but be prepared for some troubleshooting.
```

File: setup.md (hunk 1 above)
Comment (sstevens2) [reply setup.md:25]: I think the section below on using toolbox as backup if you can't get docker desktop covers it well.

**Review by sstevens2 — APPROVED** [review]
I think this looks good!  So excited to have updated install directions!  I left a couple of small suggestions on language but I'm okay with or without them.

File: setup.md (hunk 1 above)
Comment (jcohen02) [reply setup.md]: Yes, this seems reasonable, I've removed the sentence starting "Previously Windows 10 Home users..."

**Pull request merged**